addon/components/pixi-canvas.js

```diff
@@ -17,7 +17,7 @@ export default Component.extend({
   pixiRenderer: computed('width', 'height', function() {
     let { width, height } = this.getProperties('width', 'height');
 
-    return new PIXI.autoDetectRenderer(width, height);
+    return PIXI.autoDetectRenderer(width, height);
```

Comment:
@marcoow this was actually a bug since `autoDetectRenderer()` is a factory function, not a constructor